src/seedu/addressbook/data/person/Address.java

```diff
@@ -20,12 +20,12 @@
      *
```

Comment:
The PR name should contain the entire issue name, including the issue number. Just copy paste the whole thing. Your other PR had the same problem and I fixed it.